extensions/service-catalog/client/src/main/java/io/fabric8/servicecatalog/client/DefaultServiceCatalogClient.java

```diff
@@ -38,39 +38,44 @@ public DefaultServiceCatalogClient(OkHttpClient httpClient, Config configuration
     super(httpClient, configuration);
   }
 
-  public NonNamespaceOperation<ClusterServiceBroker, ClusterServiceBrokerList, ClusterServiceBrokerResource> clusterServiceBrokers() {
+  @Override
+public NonNamespaceOperation<ClusterServiceBroker, ClusterServiceBrokerList, ClusterServiceBrokerResource> clusterServiceBrokers() {
     return new ClusterServiceBrokerOperationsImpl(this.getHttpClient(), this.getConfiguration());
   }
```

Comment:
I guess this is related to #3350

Reply:
No, the IDE is just set to automatically add missing Override annotations.